include/trace_logger.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 #include <string>
 
+typedef void(*trace_callback)(int, const char*);
```

Comment:
Please include argument names on signature prototypes, it makes easier to understand what's used for.

Reply:
the has been moved to rl.net.native.